source/clear-linux/guides/telemetrics/telemetry-config.rst

```diff
@@ -0,0 +1,101 @@
+.. _telemetry-config:
+
+Telemetry Client Configuration
```

Comment:
Please revise page title to use sentence case e.g. "Telemetry client configuration"